src/networkfilemanager.cpp

```diff
@@ -1622,7 +1627,15 @@ CurlFileHandle::CurlFileHandle(PJ_CONTEXT *ctx, const char *url, CURL *handle)
 #if defined(SSL_OPTIONS)
     // https://curl.se/libcurl/c/CURLOPT_SSL_OPTIONS.html
     auto ssl_options = static_cast<long>(SSL_OPTIONS);
+    if (pj_context_get_native_ca(ctx)) {
+        ssl_options = ssl_options | CURLSSLOPT_NATIVE_CA;
```

Comment:
From CI failures, it seems that CURLSSLOPT_NATIVE_CA is only available in "recent" Curl versions, so this will need to be conditioned by a #ifdef on the Curl version

Reply:
Thank you for your clarification @rouault , I have committed a fix.